modules/installation-azure-limits.adoc

```diff
@@ -4,6 +4,7 @@
 // * installing/installing_azure/installing-azure-user-infra.adoc
 // * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
 // * installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc
+// * installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc
```

Comment:
The variable `cp` is not defined for the context `installing-restricted-networks-azure-user-provisioned`. In the preview cp is not getting replaced.

Reply:
Addressed